recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "cuda-nvcc-impl" %}
-{% set version = "12.1.105" %}
-{% set cuda_version = "12.1" %}
+{% set version = "12.2.140" %}
+{% set cuda_version = "12.2" %}
```

Comment:
Any reason not to also parse this one out of `version`?

Reply:
It would technically be possible for CUDA X.Y to ship cuda-nvcc a.b.c, unlikely in the case of nvcc, but possible. Leaving it consistent across components does smooth the update process.